diff --git a/pack/rpm.mk b/pack/rpm.mk
index 053af96..8c9e5a1 100644
--- a/pack/rpm.mk
+++ b/pack/rpm.mk
@@ -10,13 +10,7 @@ $(info Using $(RPMSPECIN) file)
 
 EXTRA_SOURCE_FILES := $(filter-out $(RPMSPECIN),$(wildcard rpm/*))
 
-RPMSPEC_AVAIL := $(shell command -v rpmspec 2> /dev/null)
-
-ifndef RPMSPEC_AVAIL
 RPMNAME := $(shell sed -n -e 's/Name:\([\ \t]*\)\(.*\)/\2/p' $(RPMSPECIN))
-else
-RPMNAME := $(shell rpmspec -P $(RPMSPECIN) | sed -n -e 's/Name:\([\ \t]*\)\(.*\)/\2/p')
-endif
 
 RPMDIST := $(shell rpm -E "%{dist}")
 PKGVERSION := $(VERSION)-$(RELEASE)$(RPMDIST)
